roadmap/implementers-guide/src/node/approval/approval-voting.md

```diff
@@ -180,6 +180,8 @@ On receiving a `CheckAndImportApproval(indirect_approval_vote, response_channel)
 
 #### `ApprovalVotingMessage::ApprovedAncestor`
 
+We record final approvals on-chain both so that we can pay the approval voters, and to map GRANDPA votes to actual finalized blocks.
```

Comment:
Omit this line

Reply:
Ok sure.  I just felt this looked like it needed clarification that it serves two roles, not one.